README.md

```diff
@@ -41,9 +41,10 @@ Please make sure you have the following general software installed
 
 | Required | Library | Version Range | Notes |
 | ------------- | ------------- | ---| --- |
-| ✔ | [Node.js](http://nodejs.com/)  | >= 6.4 | [nvm](https://github.com/creationix/nvm) is highly recommended for managing multiple node versions on a single machine |
+| ✔ | [Node.js](http://nodejs.com/)  | >= 7.10 | [nvm](https://github.com/creationix/nvm) is highly recommended for managing multiple node versions on a single machine |
 | ✔ | [Visual Studio Code](https://code.visualstudio.com/)  | >= 1.14 | We'll be using several specific features of the VS Code editor. We can't force you to use it, but you'll miss out if you don't! |
 | ✔ | [Yarn](https://yarnpkg.com/)  | >= 0.24 | An alternative to [npm](https://github.com/npm/npm) |
+| ✔ | [Firefox](https://www.mozilla.org/en-US/firefox/new/)  | >= 50 | We'll need Firefox briefly in order to create certificates. |
```

Comment:
1) Yes, let's use firefox
2) Safari provides a means to trust a certificate directly, and I plan to walk through this briefly as part of the workshop. Using HTTP/2 for development introduces the need for certs, so there's great value in touching on different debugging techniques and "certificate basics" in a class like this. I have a virtual machine with Edge ready to do the windows equivalent as well